sphinx/patterns.rst

```diff
@@ -30,20 +28,24 @@ values for these keys can be given through the command line, eg::
 Patterns kinds
 ~~~~~~~~~~~~~~~
 
-Qumin can compute various kinds of patterns. Only the ones prefixed by "patterns" can support entropy calculation.
-
-* **Affixal strategies**:
-    * ``endings``: Removes the longest common initial string for each row.
-    * ``endingsPairs``: Endings, tabulated as pairs for all combinations of columns.
-    * ``endingsDisc``: Discontinuous endings. Removes the longest common substring, left aligned
-* **Alternations**:  Alternations have no contextes. These were used for comparing macroclass strategies on French and European Portuguese.
-    * ``globalAlt``: As ``EndingsDisc``, tabulated as pairs for all combinations of columns.
-    * ``localAlt``:  Inferred from pairs of cells, left aligned.
-* **Patterns**: Binary alternation Patterns. All patterns have alternations and generalized contexts. Various alignment strategies are offered for comparison. Arbitrary number of changes supported.
-    * ``patternsLevenshtein``: Aligned with simple edit distance.
-    * ``patternsPhonsim``: Aligned with edit distances based on phonological similarity.
-    * ``patternsSuffix``: Fixed left alignment, only interesting for suffixal languages.
-    * ``patternsPrefix``: Fixed right alignment, only interesting for prefixal languages.
-    * ``patternsBaseline``: Baseline alignment, follows Albright & Hayes 2002. A single change, with a priority order: Suffixation > Prefixation > Stem-internal alternation (ablaut/infixation)
-
-Most of these were implemented for comparison purposes during Sacha Beniamine's PhD. It is recommended to use the default `patternsPhonsim` in most cases. To avoid relying on your phonological features files for alignment scores, use `patternsLevenshtein`. Only these two are full patterns with generalization both in the context and alternation.
+Qumin can compute various kinds of patterns that can be used for entropy calculations. They have alternations and generalized contexts:
+
+* ``patternsLevenshtein``: Aligned with simple edit distance.
+* ``patternsPhonsim``: Aligned with edit distances based on phonological similarity.
+
+It is recommended to use the default `patternsPhonsim` in most cases. To avoid relying on your phonological features files for alignment scores, use `patternsLevenshtein`. Only these two are full patterns with generalization both in the context and alternation.
+
+.. warning::
+    Additional strategies were implemented for comparison purposes during Sacha Beniamine's PhD. They are now deprecated but are available until Qumin 2.0.1.
```

Comment:
"Support for these has now been discontinued." => and remove the detailed list. (since the code _is_ removed below !)